lib/generators/blacklight_marc/templates/app/models/marc_indexer.rb

```diff
@@ -0,0 +1,150 @@
+$:.unshift './config/SolrMarc'
```

Comment:
Traject is (at least for now, afaik) insistent on looking the property files up on the classpath, and that's the legacy directory.